### ISSUE TITLE
Text direction not working anymore in PDF export

### ISSUE THREAD

**Issue by timoludwig** [digitalfabrik/integreat-cms#1954]
### Describe the Bug

Somehow RTL languages are printed LTR in the PDF export

### Steps to Reproduce

1. Go to a right-to-left written language, e.g. https://integreat.app/kreishoexter/ar/%D8%A7%D9%84%D8%AD%D9%8A%D8%A7%D8%A9-%D8%A7%D9%84%D9%8A%D9%88%D9%85%D9%8A%D8%A9/%D8%A7%D9%84%D8%B3%D9%83%D9%86/%D8%A7%D9%84%D9%84%D9%88%D8%A7%D8%B2%D9%85-%D8%A7%D9%84%D8%A3%D8%B3%D8%A7%D8%B3%D9%8A%D8%A9-%D9%84%D8%AA%D8%AC%D9%87%D9%8A%D8%B2-%D8%B4%D9%82%D8%A9
2. Export as PDF
3. See that the text is written left-to-right

### Expected Behavior

RTL languages should be exported in RTL

### Actual Behavior

RTL languages are exported in LTR

### Additional Information

![Screenshot (862)](https://user-images.githubusercontent.com/16021269/206442583-f38aca8a-472b-46fe-bd40-ab3571910931.png)

**Comment by timoludwig** [comment]
Our [Arabic test PDF](https://github.com/digitalfabrik/integreat-cms/blob/develop/tests/pdf/files/3b02f5ea5b/Integreat%20-%20Arabisch%20-%20%D9%85%D8%B9%D9%84%D9%88%D9%85%D8%A7%D8%AA%20%D8%A7%D9%84%D9%88%D8%B5%D9%88%D9%84.pdf) looks good, so not sure whether this problem only occurs in some special cases.

**Comment by seluianova** [comment]
> Our Arabic test PDF looks good

I haven't dealt with Arabic languages before, so I'm not sure about my findings. But here they are:

Titles in the table content have a different direction, than in the CMS:
![image](https://user-images.githubusercontent.com/115008338/207369833-3fdceaf6-5d36-42ea-89ad-aaaf3de6e66d.png)

Then, on the page itself, the title looks correct to me, but the content has the wrong direction:
![image](https://user-images.githubusercontent.com/115008338/207371308-be08e981-7c7d-40df-8f17-a77d1bd5adf6.png)

Basically the same situation as here: https://integreat.app/kreishoexter/ar/%D8%A7%D9%84%D8%AD%D9%8A%D8%A7%D8%A9-%D8%A7%D9%84%D9%8A%D9%88%D9%85%D9%8A%D8%A9/%D8%A7%D9%84%D8%B3%D9%83%D9%86/%D8%A7%D9%84%D9%84%D9%88%D8%A7%D8%B2%D9%85-%D8%A7%D9%84%D8%A3%D8%B3%D8%A7%D8%B3%D9%8A%D8%A9-%D9%84%D8%AA%D8%AC%D9%87%D9%8A%D8%B2-%D8%B4%D9%82%D8%A9

**Comment by seluianova** [comment]
> Then, on the page itself, the title looks correct to me, but the content has the wrong direction

This part was changed within the commit:
https://github.com/digitalfabrik/integreat-cms/commit/706438671712859a253ca7f0d0b236c8fb97a7d6

**Comment by seluianova** [comment]
Text direction is interpreted incorrectly because of 'safe' template tag:
`{{ page_translation.content|pdf_strip_fontstyles|safe }}`
